build/src/root/insurance/application.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
Not 100% familar with these sort of idioms. Are these properties and `Object.defineProperty(exports, "__esModule", { value: true });` for backwards compatability?

Reply:
Ah. See you've got typescript in the project and this is the compiled output. Is it typical to check the compiled output in?